ledger/src/blockstore_processor.rs

```diff
@@ -1410,26 +1403,43 @@ fn load_frozen_forks(
 
     let on_halt_store_hash_raw_data_for_debug = opts.on_halt_store_hash_raw_data_for_debug;
     if Some(bank_forks.read().unwrap().root()) != opts.halt_at_slot {
+        let recyclers = VerifyRecyclers::default();
+        let mut all_banks = HashMap::new();
+
+        const STATUS_REPORT_INTERVAL: Duration = Duration::from_secs(2);
```

Comment:
nit: placement of the const seems weird to me. I think it makes sense to either put it at the top of the function, or closer to where it's actually used. Right now it seems like it's just in a random scope haha

Reply:
The placement was to be adjacent to `last_status_report` instant, and all of the other variables that get updated in the loop. Could throw it right above the `if` statement where it is read if you think that makes more sense.

Reply:
Got it, I think that makes sense with that justification. Not too strongly opinionated about it.